RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  ManageSubscriptionsView.swift
+//
+//
+//  Created by Andrés Boedo on 5/3/24.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct ManageSubscriptionsView: View {
+
+    @Environment(\.openURL)
+    var openURL
+
+    @StateObject
+    private var viewModel = ManageSubscriptionsViewModel()
+
+    init() { }
+
+    init(viewModel: ManageSubscriptionsViewModel) {
```

Comment:
Is this initializer solely for enabling previews and possibly mocks when testing? We could make this method `fileprivate` to keep its interface cleaner. (This may apply to other classes too.)

```suggestion
    fileprivate init(viewModel: ManageSubscriptionsViewModel) {
```

Reply:
yes, only for previews and tests